upgrade/v2.5/script/upgrade.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+
+# Set the ENV var for KDD backend
+export DATASTORE_TYPE=kubernetes
+
+set -e
+
+# Create the CRDs by applying the crds.yaml manifest.
+echo "Creating Calico CRDs..."; echo
+/sbin/kubectl apply -f crds.yaml
+
+# Get IPPools list with old calicoctl (from TPR) and save it as a yaml file.
+# Apply the yaml file with the new calicoctl (to CRD)
+echo "Migrating IPPools..."
+/sbin/calicoctl-v1.4 get ippool -o yaml > ippool.yaml
+retval=$?
+if [ $retval == 0 ]; then
+    if [ -f ippool.yaml ]; then
+        echo "Successfully got the IPPools:"
+        cat ippool.yaml
+        lines=`cat ippool.yaml | wc -l`
+        cat ippool.yaml | grep '\[\]'
+        retval=$?
+        if [ $retval == 0 -a $lines == 1 ]; then
+            echo "No IPPools found to migrate."
+        else
+            /sbin/calicoctl-v1.5 apply -f ippool.yaml
+        fi
+    else
+       echo "No IPPools found to migrate."
+    fi
+else
+    echo "Failed to get IPPools"
+fi
+
+# Get BGPPeers list with old calicoctl (from TPR) and save it as a yaml file.
+# Apply the yaml file with the new calicoctl (to CRD)
+echo; echo "Migrating BGPPeers..."
```

Comment:
nope, kubelet logs don't do anything special with pod logs, it gets printed as it is